config.js

```diff
@@ -124,7 +124,7 @@ const conf = convict({
   }
 })
 
-var envConfig = path.join(__dirname, '..', 'config', conf.get('env') + '.json')
+var envConfig = path.join(__dirname, 'config', conf.get('env') + '.json')
```

Comment:
Was this a straight-up bug in the previous version, looking for the files in the wrong place?